portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -35,14 +35,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         if (userService.isUserLoggedIn()) {
             String logoutUrl = userService.createLogoutURL(urlToRedirectTo);
```

Comment:
We have both logoutUrl and createLogoutURL here, could you make them consistent in the file?
Since URL is a well-known short form, usually we use logoutURL as var name in this case.
Same for loginUrl below.

Reply:
Of course! The variable names are updated! :)